pkg/services/alerting/notifiers/wecom.go

```diff
@@ -0,0 +1,135 @@
+package notifiers
```

Comment:
Apologies if I didn't mention this before, but I don't think we should support the old alerting. We can remove this notifier and stick to the one in ngalert.